src/main/java/nextstep/subway/line/domain/Line.java

```diff
@@ -20,6 +20,7 @@ public class Line extends BaseEntity {
     private String color;
     @Embedded
     Sections sections = new Sections();
```

Comment:
```suggestion
    private Sections sections = new Sections();
```

PR 올리기 전에 혹시 빼먹은거 있나 한번 점검해주세요

Reply:
주의하겠습니다